rules/cloud/azure_virtual_network_modified_or_deleted.yml

```diff
@@ -0,0 +1,24 @@
+title: Azure Virtual Network Modified or Deleted
+id: bcfcc962-0e4a-4fd9-84bb-a833e672df3f
+description: Identifies when a Virtual Network is modified or deleted in Azure.
+author: Austin Songer @austinsonger
+status: experimental
+date: 2021/08/08
+references:
+    - https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations
+logsource:
+  service: azure.activitylogs
+detection:
+    selection:
+        properties.message: 
+            - MICROSOFT.NETWORK/VIRTUALNETWORKGATEWAYS/WRITE
+            - MICROSOFT.NETWORK/VIRTUALNETWORKGATEWAYS/DELETE
+            - MICROSOFT.NETWORK/VIRTUALNETWORKS/WRITE
+            - MICROSOFT.NETWORK/VIRTUALNETWORKS/DELETE
+    condition: selection
```

Comment:
there are many 'sub key' under  so I think more to
```yaml
properties.message|startswith:
    - MICROSOFT.NETWORK/VIRTUALNETWORKGATEWAYS/
    - MICROSOFT.NETWORK/VIRTUALNETWORKS/
properties.message|endswith:
   - /WRITE
   - /DELETE
```

Reply:
I will do this for the others that I'm working on.